pkg/cloudstorage_fs.go

```diff
@@ -161,7 +179,7 @@ func (c *cloudStorageFS) client(ctx context.Context, scope string, extraScopes .
 		return nil, nil, errors.Wrap(err, "cloud storage: unable to retrieve default token source")
 	}
 
-	client, err := gcp.NewHTTPClient(gcp.DefaultTransport(), gcp.CredentialsTokenSource(creds))
+	client, err := gcp.NewHTTPClient(gcp.DefaultTransport(), creds.TokenSource)
```

Comment:
Was `gcp.CredentialsTokenSource(creds)` doing anything special? I think it's just returning a type wrapper of creds.TokenSrouce?